TemplateBicep/main.bicep

```diff
@@ -0,0 +1,66 @@
+param location string = resourceGroup().location
+param uniqueSolutionPrefix string
+param discoveryZipUrl string
```

Comment:
As far as I can see discoveryZip Url should be static variable, it is tied to the version

Reply:
https://github.com/Azure/iotedge-lorawan-starterkit/blob/dev/Template/azuredeploy.json#L96

Reply:
Changes added. Please review.